### DISCUSSION TITLE
550-Unknown user 550 Sender verify failed

### DISCUSSION THREAD

**Discussion by GRR-ZA** [postalserver/postal#1848] [Help with using Postal] [answered]
Hey all,

I've setup a new postal server which has been working for most outbound mail but there are a few hard fails and soft fails as per below.

550-Verification failed for <k66dha@rp.postal.xxxxxxxxxxxx.com> 550-Unknown user 550 Sender verify failed

450 4.1.8 <k66dha@rp.postal.xxxxxxxxxxxx.com>: Sender address rejected: Domain not found

What I'm struggling to understand is where it's getting the sender address from.

Any advise would be appreciated.

**Comment by willpower232** [comment]
Where are you seeing those errors, can you include a screenshot?

**Reply by bonest** [reply]
I have the same errors on the postal admin interface when sending to a domain when I use postalserver as a transactional server to send the email to a mailserver that holds the inbox for the domain. 

Its kind of difficult to explain

![image](https://github.com/postalserver/postal/assets/15220746/dc6f6bb0-cc65-43c8-bdf8-28783066ee7c)

Postfix has this explanation.

reject_unknown_sender_domain
Reject the request when Postfix is not the final destination for the sender address, and the MAIL FROM domain has 1) no DNS MX and no DNS A record, or 2) a malformed MX record such as a record with a zero-length MX hostname (Postfix version 2.3 and later).
The reply is specified with the [unknown_address_reject_code](https://www.postfix.org/postconf.5.html#unknown_address_reject_code) parameter (default: 450), [unknown_address_tempfail_action](https://www.postfix.org/postconf.5.html#unknown_address_tempfail_action) (default: [defer_if_permit](https://www.postfix.org/postconf.5.html#defer_if_permit)), or 550 (nullmx, Postfix 3.0 and later). See the respective parameter descriptions for details.

**Reply by bonest** [reply, marked as answer]
I found the problem. Inspired by this thread https://github.com/postalserver/postal/issues/653

It was a typo in the a Record for the rp.mail.example.com A record. I had it created as rp.mail.example.com.example.com. Fixing that A record solved my problem.
